Mathlib/GroupTheory/FreeGroup.lean

```diff
@@ -991,82 +1009,83 @@ variable {β : Type u}
 
 @[to_additive]
 instance : Monad FreeGroup.{u} where
-  pure α := of
-  map α β f := map f
-  bind α β x f := lift f x
+  pure {_α} := of
+  map {_α} {_β} {f} := map f
+  bind {_α} {_β} {x} {f} := lift f x
 
 @[elab_as_elim, to_additive]
 protected theorem induction_on {C : FreeGroup α → Prop} (z : FreeGroup α) (C1 : C 1)
     (Cp : ∀ x, C <| pure x) (Ci : ∀ x, C (pure x) → C (pure x)⁻¹)
     (Cm : ∀ x y, C x → C y → C (x * y)) : C z :=
   Quot.inductionOn z fun L =>
-    List.recOn L C1 fun ⟨x, b⟩ tl ih => Bool.recOn b (Cm _ _ (Ci _ <| Cp x) ih) (Cm _ _ (Cp x) ih)
+    List.recOn L C1 fun ⟨x, b⟩ _tl ih => Bool.recOn b (Cm _ _ (Ci _ <| Cp x) ih) (Cm _ _ (Cp x) ih)
 #align free_group.induction_on FreeGroup.induction_on
-#align free_add_group.induction_on freeAddGroup.induction_on
+#align free_add_group.induction_on FreeAddGroup.induction_on
 
-@[simp, to_additive]
+@[to_additive]
```

Comment:
If you remove a simp attribute, please leave a porting note explaining why (a 1 line summary of the linter complaint).